tests/client/asynchronous_test.py

```diff
@@ -49,3 +54,76 @@ def test_matches(self):
     def assert_response_type(self, matcher, path, expected_response_type):
         response = json.loads(matcher.response(path))
         assert response["response-type"] == expected_response_type
+
+
+class TestResolveHost:
```

Comment:
You don't need a class here, the function itself is enough (just remove self)

Reply:
Addressed in https://github.com/elastic/rally/pull/1707/commits/fe762dfdc66ad80bec07fbcb3a0d4be7eec11a64